micro-rdk/.cargo/config.toml

```diff
@@ -17,10 +17,3 @@ rustflags = ["-C", "default-linker-libraries"]
 linker = "ldproxy"
 runner = "espflash --monitor"
 rustflags = ["-C", "default-linker-libraries"]
-
-
-
-[env]
```

Comment:
It was not clear to me that these had / have any effect. I think the ones in `micro-rdk-server` were the only ones that matter.